pages/reference/diagnostics.md

```diff
@@ -0,0 +1,67 @@
+# Check Descriptions
+
+As part of the diagnostics suite, you will find various checks that can be run on-device. Below is a description of each
```

Comment:
@xginn8 these check all get run together right? or are we making it so a single command can be run?

Reply:
You are correct, that is strangely worded.